pkg/health/client/client.go

```diff
@@ -33,10 +33,27 @@ import (
 	"github.com/go-openapi/strfmt"
 )
 
+type ConnectivityStatusType int
```

Comment:
nit: The description of this commit contains a typo: `uknkown`

Reply:
Fixed, thanks!